simapp/app_legacy.go

```diff
@@ -430,6 +433,14 @@ func NewSimApp(
 	app.configurator = module.NewConfigurator(app.appCodec, app.MsgServiceRouter(), app.GRPCQueryRouter())
 	app.ModuleManager.RegisterServices(app.configurator)
 
+	autocliv1.RegisterQueryServer(app.GRPCQueryRouter(), runtimeservices.NewAutoCLIQueryService(app.ModuleManager.Modules))
+
+	reflectionSvc, err := runtimeservices.NewReflectionService()
```

Comment:
Let's maybe add a quick note in upgrading.md for v0.47

Reply:
cc @ilgooz for visibility on this feature